src/Frontend/Page/Idea.hs

```diff
@@ -68,7 +68,11 @@ instance ToHtml ViewIdea where
     -- The result might be that too much information is displayed.
     toHtml p@(ViewIdea idea phase) = semanticDiv p $ do
         header_ [class_ "detail-header"] $ do
-            a_ [class_ "btn m-back detail-header-back", href_ U.Broken] "Zum Thema"
+            a_ [ class_ "btn m-back detail-header-back"
```

Comment:
Fair enough. There is a couple more such occurrences, so that's probably not a typo:
```
src/Frontend/Page/Admin.hs:                    a_ [href_ U.Broken, class_ "btn-cta"] "Passwort zurücksetzen" >> br_ []
src/Frontend/Page/Admin.hs:                    a_ [href_ U.Broken, class_ "btn-cta"] "Nutzer löschen" >> br_ []
```